config/database.yml

```diff
@@ -24,9 +24,14 @@ defaults: &defaults
 development:
   primary:
     <<: *defaults
+    migrations_paths: db/primary_migrate
```

Comment:
the default is `db/migrate` but specifying this makes sure we don't run the old migrations in the jobs-only DB, complete separation of DBs